docs/api/other-api/adapter.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "@react-router/{adapter}"
+---
+
+# Server Adapters
+
+## Official Adapters
+
+Idiomatic React Router apps can generally be deployed anywhere because React Router adapts the server's request/response to the [Web Fetch API][web-fetch-api]. It does this through adapters. We maintain a few adapters:
+
+- `@react-router/architect`
```

Comment:
oof, I don't even want to mention this one 🙈

Almost wonder if it's worth also suggestion people checkout [SST](https://sst.dev/docs/start/aws/react/)

Though at that point, we probably have to do it for every third party adapter, which is not really the point here 🤔

Reply:
I think as long as we're publishing it we should probably include it :/.  If we want to straight up leave it out I think we'd be better off officially deprecating it or something.

And yeah linking to third parties is always a slippery slope to avoid any perceived favoritism...

Reply:
Yeah, that's fine. But I do think we should consider deprecating it soon, or signal that we're going to deprecate it or something

[it seems to barely be used](https://npmtrends.com/@react-router/architect-vs-@remix-run/architect) btw

Reply:
ok, you convinced me lol - [0.3% of RR framework mode](https://npmtrends.com/@react-router/architect-vs-@react-router/cloudflare-vs-@react-router/dev) users are using it so maybe reference docs are enough for them